easybuild/easyblocks/q/quantumespresso.py

```diff
@@ -73,16 +73,78 @@ def patch_step(self):
     def configure_step(self):
         """Custom configuration procedure for Quantum ESPRESSO."""
 
+        # compose list of DFLAGS (flag, value, keep_stuff)
+        # for guidelines, see include/defs.h.README in sources
+        dflags = []
+
+        repls = []
+
+        extra_libs = []
+
+        comp_fam_dflags = {
+            toolchain.INTELCOMP: '-D__INTEL',
+            toolchain.GCC: '-D__GFORTRAN -D__STD_F95',
+        }
+        dflags.append(comp_fam_dflags[self.toolchain.comp_family()])
+
         if self.toolchain.options.get('openmp', False) or self.cfg['hybrid']:
             self.cfg.update('configopts', '--enable-openmp')
+            dflags.append(" -D__OPENMP")
 
         if not self.toolchain.options.get('usempi', None):
```

Comment:
@akesandgren Positive logic is easier to follow, so please change and switch if/else bodies?

```python
if self.toolchain.options.get('usempi', None):
    ...
else:
    ...
```

Same suggestion for other places you're adding an `else`